nixos/tests/uptime-kuma.nix

```
@@ -1,10 +1,8 @@
 import ./make-test-python.nix ({ lib, ... }:
 
-with lib;
-
 {
   name = "uptime-kuma";
-  meta.maintainers = with maintainers; [ julienmalka ];
+  meta.maintainers = [ lib.maintainers.julienmalka ];
```

```suggestion
  meta.maintainers = with lib.maintainers; [ julienmalka ];
```
When there are multiple maintainers we would end up with code duplication.
I don't see a reason why a meta.maintainers with only one entry should be differently formatted then one with multiple, hence I am suggesting to make it ready for multiple maintainers and don't suggest code duplication.